picard/__init__.py

```diff
@@ -83,7 +82,11 @@ def crash_handler():
     # with minimum chance to fail.
     from tempfile import NamedTemporaryFile
     import traceback
-    trace = traceback.format_exc()
+    if exc:
+        # Since Python 3.10 calling only traceback.format_exception(exc) is possible
+        trace = "\n".join(traceback.format_exception(type(exc), exc, exc.__traceback__))
```

Comment:
`"\n"` here causes an extra newline, https://docs.python.org/3.10/library/traceback.html#traceback.format_exception says: "The return value is a list of strings, each ending in a newline and some containing internal newlines."

So I think we can just `"".join()` here.

Reply:
You are right. This is also what `traceback.format_exc`  does internally.